com.unity.ml-agents.extensions/Documentation~/Grid-Sensor.md

```diff
@@ -23,7 +23,7 @@ This is simple to implement and provides enough information for most simple game
 
 **Camera**
 
-The Camera provides the agent with either a grayscale or an RGB image of the game environment. It goes without saying that there non-linear relationships between nearby pixels in an image. It is this intuition that helps form the basis of Convolutional Neural Networks (CNNs) and established the literature of designing networks that take advantage of these relationships between pixels. Following this established literature of CNNs on image based data, the MLAgent's Camera Sensor provides a means by which the agent can include high dimensional inputs (images) into its observation stream.
+The Camera provides the agent with either a grayscale or an RGB image of the game environment. It goes without saying that there's non-linear relationships between nearby pixels in an image. It is this intuition that helps form the basis of Convolutional Neural Networks (CNNs) and established the literature of designing networks that take advantage of these relationships between pixels. Following this established literature of CNNs on image based data, the MLAgent's Camera Sensor provides a means by which the agent can include high dimensional inputs (images) into its observation stream.
```

Comment:
```suggestion
The Camera provides the agent with either a grayscale or an RGB image of the game environment. It goes without saying that there is a non-linear relationships between nearby pixels in an image. It is this intuition that helps form the basis of Convolutional Neural Networks (CNNs) and established the literature of designing networks that take advantage of these relationships between pixels. Following this established literature of CNNs on image based data, the MLAgent's Camera Sensor provides a means by which the agent can include high dimensional inputs (images) into its observation stream.
```

I corrected a typo here, but I think this sentence in general is confusing. The reason that CNNs are useful is not that pixels have a non-linear relationship, but that there are spatial invariances across images which can be taken advantage of by re-using the same kernel in different parts of the image.